src/components/Topics.jsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom"
+import { getArticles } from "../../api";
+import SingleArticle from "./SingleArticle";
+
+
+
+
+const Topic = ()=>{
+const topic = useParams();
+const [isLoading, setIsLoading] =useState(true)
+const [articles, setArticles] =useState([])
+
+    useEffect(()=>{
+        setIsLoading(true)
```

Comment:
As this state is created with the value of true, we don't need to set it to true again here. This only may be needed if we are error handling. Which leads me onto error handling - could we add some in :)